index.md

```diff
@@ -16,12 +17,15 @@ Given a version number MAJOR.MINOR.PATCH, increment the:
    manner, and
 1. PATCH version when you make backwards-compatible bug fixes.
 
-Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
+Additional labels for pre-release and build metadata are available as extensions
+to the MAJOR.MINOR.PATCH format.
 
-Introduction
+<a name="intro"></a>
+# Introduction
 ------------
+<sup>[[link](#intro)]</sup>  
 
-In the world of software management there exists a dread place called
+In the world of software management there exists a dreaded place called
```

Comment:
Could you remove any changes unrelated to adding anchors to this PR? The text should exactly match what's currently on http://semver.org.